openhands/runtime/action_execution_server.py

```diff
@@ -198,6 +200,8 @@ def __init__(
         self.start_time = time.time()
         self.last_execution_time = self.start_time
         self._initialized = False
+        self.downloaded_files: list[str] = []
+        self.downloads_directory = '/workspace/downloads'
```

Comment:
This folder might not exist at all (in GAIA eval harness we explicitly create this folder). Whatever folder path we use, we should also keep both docker runtime and local runtime in mind.

@xingyaoww Do you have some idea here? I was going to suggest a new config option until I see a recent refactoring of yours (https://github.com/All-Hands-AI/OpenHands/pull/8242) that deprecates that approach.

Reply:
When running locally, got the following error
`playwright._impl._errors.Error: BrowserType.launch: EACCES: permission denied, mkdir '/workspace'`

How about `/tmp/.downloads`?